packages/bot-utils/src/checks.ts

```diff
@@ -0,0 +1,28 @@
+import Mangrove from "@mangrovedao/mangrove.js";
+import { CommonLogger } from "./logging/coreLogger";
+
+const FRESHNESS_BLOCK_SECONDS = 60 * 5; // TODO: should be more precise
+const HOW_MANY_BLOCK_BACK = 10;
+
```

Comment:
Could we make these to configurable?

Reply:
Yes we can but this is I hope a temporary measure, so do we want it ?

Reply:
I'm planning for a better version that requires a RES change I will update you shortly

Reply:
I was planning to add block timestamp inside the Block struct of reliable-events-subscriber but it's not possible as we don't have access to this info inside a multi call. I also started to look at what we currently we have in the mangrove.js config and we have `estimatedBlockTimeMs` we could use that and multiply it to a factor in config.